src/components/parallex.jsx

```diff
@@ -0,0 +1,30 @@
+window.addEventListener("scroll", function () {
+  const distance = window.scrollY;
+
+  let vid = document.getElementById("bg-video");
```

Comment:
Without going too deep into React, so for now it's fine so you don't have to change it, but whenever you're using the [Document Object Model](https://www.w3schools.com/whatis/whatis_htmldom.asp) directly, which is what you're doing with `document.querySelector`, you're not using React but old school Javascript. React is kind of like an environment where you don't need to interact with the DOM. You're working with React instead and React then tells the DOM what to update. You're able to bypass React, but most times you don't need to and there's a React alternative. Here it would be adding classnames conditionally like I saw before with [ternary statements](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator). So from now on when you're using something that starts with `document`, it might not be the React way. Why don't you wanna do that? Because eventually everything is quicker and more performant. Anyway long story but, best practice is to avoid using DOM directly unless really (and rarely) impossible.

Reply:
This I put on my list as well, to make it more proper, but works for now.